mesonbuild/modules/gnome.py

```diff
@@ -1930,6 +1937,7 @@ def _make_mkenum_impl(
             extra_depends=depends,
             # https://github.com/mesonbuild/meson/issues/973
             absolute_paths=True,
+            description='Generating GObject enum files {}',
```

Comment:
There's just one of these, right?

```suggestion
            description='Generating GObject enum file {}',
```

Reply:
I don't know... I just based it on the documentation:

> Generates enum files for GObject using the glib-mkenums tool.